packages/react-app/src/components/global/Header/UserMenu.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+	Avatar,
+	Button,
+	Input,
+	Menu,
+	MenuButton,
+	MenuItem,
+	MenuList,
+	Modal,
+	ModalBody,
+	ModalContent,
+	ModalHeader,
+	ModalOverlay,
+} from '@chakra-ui/react';
+import { useSession } from 'next-auth/react';
+import { useDisclosure } from '@chakra-ui/react';
+import { toggleDiscordSignIn } from '@app/services/discord.service';
+import React from 'react';
+
+const UserMenu = () => {
+	const { data: session } = useSession({ required: false });
+	const { isOpen, onOpen, onClose } = useDisclosure();
+	return (
+		<Menu>
+			<MenuButton as={'button'}>
+				<Avatar src={session?.user?.image ?? ''} mr={'.25em'} size={'md'} />
```

Comment:
If the avatar is not found, would it be better to render a placeholder, or not render the avatar at all

Reply:
I'll add a comment here but the Avatar loads a default blank user pic when the discord user image doesn't exist or fails to load.